orchagent/main.cpp

```diff
@@ -349,6 +350,7 @@ int main(int argc, char **argv)
     string record_location = ".";
     string swss_rec_filename = "swss.rec";
     string sairedis_rec_filename = "sairedis.rec";
+    string zmq_server_address = "tcp://*:1234";
```

Comment:
Should we define the port to schema or /etc/sonic/constants.yml?

Reply:
Will create another PR to put this to sonic-swss-common schema.h and update later.